components/batch-system/src/router.rs

```diff
@@ -38,6 +52,8 @@ pub struct Router<N: Fsm, C: Fsm, Ns, Cs> {
     pub(crate) normal_scheduler: Ns,
     control_scheduler: Cs,
 
+    /// Count of Mailbox that is not destroyed.
+    state_cnt: Arc<AtomicUsize>,
```

Comment:
What's the different between `state_cnt` and `alive_cnt`?

Reply:
`state_cnt ` is modified when a mailbox created and destroyed.`alive_cnt` is modified when a mailbox poped and pushed from hashmap.
They could be different since we have a mailbox cache.